src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -250,6 +250,18 @@ public PackageBuilder CreateBuilder(string basePath, NuGetVersion version, strin
             }
 
             var projectAuthor = InitializeProperties(builder);
+
+            // Only override properties from assembly extracted metadata if they haven't 
+            // been specified also at construction time for the factory (that is, 
+            // console properties always take precedence.
+            foreach (var key in builder.Properties.Keys)
+            {
+                if (!_properties.ContainsKey(key) &&
+                    !ProjectProperties.ContainsKey(key))
+
+                    _properties.Add(key, builder.Properties[key]);
```

Comment:
Our coding standard is to put curly braces around all blocks even if they are only one line long.  Please put them here and other places in your PR.